src/3d/qgs3dsceneexporter.cpp

```diff
@@ -523,7 +527,65 @@ QVector<Qgs3DExportObject *> Qgs3DSceneExporter::processSceneLoaderGeometries( Q
 Qgs3DExportObject *Qgs3DSceneExporter::processGeometryRenderer( Qt3DRender::QGeometryRenderer *geomRenderer, const QString &objectNamePrefix, float sceneScale, QVector3D sceneTranslation )
 {
   // We only export triangles for now
-  if ( geomRenderer->primitiveType() != Qt3DRender::QGeometryRenderer::Triangles ) return nullptr;
+  if ( geomRenderer->primitiveType() != Qt3DRender::QGeometryRenderer::Triangles )
+    return nullptr;
+
+  Qt3DQGeometry *geometry = geomRenderer->geometry();
+
+  // TODO: TO REMOVE
+  qDebug() << "============= Will export geom" << geomRenderer->parent()->objectName();
+
+  // In polygon cases, we have multiple feature geometries within the same geometry object (QgsTessellatedPolygonGeometry).
+  // The QgsTessellatedPolygonGeometry class holds the list of all feature ids included.
+  // To avoid to export the same geometry (can be included in multiple QgsTessellatedPolygonGeometry) more than once,
```

Comment:
"To avoid to export the same geometry (can be included " => "To avoid exporting the same geometry (it can be included